.travis.yml

```diff
@@ -6,8 +6,6 @@ branches:
 
 language: ruby
 rvm:
-  - 2.0.0
```

Comment:
The config material doesn't work correctly with Ruby versions lower than 2.2. This is known issue, and @matthewrkrieger was planning on trying to figure out why that is.